app/poros/forecast.rb

```diff
@@ -33,8 +33,8 @@ def get_daily_forecast(data)
     end
   end
 
-  def convert_to_miles(meters)
-    # 1 mile = 1609.34 meters
-    meters / 1609.34
-  end
+  # def convert_to_miles(meters)
+  #   # 1 mile = 1609.34 meters
+  #   meters / 1609.34
+  # end
```

Comment:
Ideally if someone on the FE was creating, this method would be available. But not knowing the semantics of the FE, I'm not sure if this logic stays here because it's technically presentation logic or if FE has a way to do calculations as well.